src/search/axioms.cc

```diff
@@ -60,23 +61,39 @@ AxiomEvaluator::AxiomEvaluator(const TaskProxy &task_proxy) {
             axiom_literals.emplace_back(var.get_domain_size());
 
         // Initialize rules
+        int count = 0;
+        // Since we are skipping some ops, we cannot access them through their
+        // id position directly
+        unordered_map<int, size_t> axiom_id_to_position;
```

Comment:
I think we often use a vector<int> for this situation, initialized with -1 for all axioms.

Reply:
And I'd store int instead of size_t values.